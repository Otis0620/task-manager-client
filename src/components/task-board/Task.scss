.task {
  margin-top: 15px;
  box-shadow: gray;
  background-color: white;
  box-shadow: 4px 2px 11px 0px rgba(197, 197, 197, 1);
}

.task-header {
  display: flex;
  font-size: .5em;
  word-wrap: break-word;
  color: white;

  p {
    padding-left: 5px
  }
}