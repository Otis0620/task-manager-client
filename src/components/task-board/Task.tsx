import './Task.scss';
import React, { FC } from 'react';
import { TaskPropsI } from './interfaces';

export const Task: FC<TaskPropsI> = ({ backgroundColor, taskNumber, taskTitle }) => (
  <div className="task">
    <div className="task-header" style={{ backgroundColor: backgroundColor }}>
      <p>{taskNumber}</p>

      <p>{taskTitle}</p>
    </div>

    <div>
      <p>sdf</p>
    </div>
  </div>
);
