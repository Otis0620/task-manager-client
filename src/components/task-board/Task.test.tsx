import React from 'react';
import { shallow } from 'enzyme';
import { Task } from './Task';

test('Task', () => {
  const wrapper = shallow(
    <Task backgroundColor="red" taskNumber={1} description="..." taskTitle="Fake Title" />,
  );

  expect(wrapper).toMatchSnapshot();
});
