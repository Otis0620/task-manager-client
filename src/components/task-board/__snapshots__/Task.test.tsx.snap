// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Task 1`] = `
<div
  className="task"
>
  <div
    className="task-header"
    style={
      Object {
        "backgroundColor": "red",
      }
    }
  >
    <p>
      1
    </p>
    <p>
      Fake Title
    </p>
  </div>
  <div>
    <p>
      ...
    </p>
  </div>
</div>
`;
